### DISCUSSION TITLE
503 Service Temporarily Unavailable

### DISCUSSION THREAD

**Discussion by ekoyuliawan** [kirimwa-id/forum#177] [Tanya Jawab] [answered]
Dear mas @rioastamal, mohon bantuannya. Baru saja terjadi error 503 Service Temporarily Unavailable. Error yang sama apabila mencoba untuk melakukan scan ulang. Terima kasih atas bantuannya.

**Comment by rioastamal** [comment, marked as answer]
Thanks udah report. Sudah fixed dari kemarin. 

Server yang bermasalah adalah yang free, pengguna enterprise tidak terpengaruh.
